rosbag2_transport/test/rosbag2_transport/rosbag2_play_test_fixture.hpp

```diff
@@ -30,7 +30,7 @@ class RosBag2PlayTestFixture : public Rosbag2TransportTestFixture
     sub_ = std::make_shared<SubscriptionManager>();
   }
 
-  ~RosBag2PlayTestFixture() override
+  virtual ~RosBag2PlayTestFixture()
```

Comment:
why this change?

Reply:
Oh - I'll change it back, I wasn't thinking very hard about it. I see that declaring `override` _forces_ the parent class to also have a virtual destructor. I just wasn't used to seeing that pattern.